docs/src/user-guide/wallet.md

```diff
@@ -0,0 +1,61 @@
+# The Anoma Wallet
+
+The Anoma wallet allows you to store and use addresses and keys by their alias.
+
+The wallet's state is stored under `.anoma/wallet.toml` (with the default `--base-dir`), which will be created for you if it doesn't already exist when you run any command that accesses the wallet. A newly created wallet will be pre-loaded with some default addresses.
```

Comment:
The wallet is being updated to be stored in the chain sub-dir. In public release docs, we can replace it with the chosen chain ID.

```suggestion
The wallet's state is stored under `.anoma/{chain_id}/wallet.toml` (with the default `--base-dir`), which will be created for you if it doesn't already exist when you run any command that accesses the wallet. A newly created wallet will be pre-loaded with some default addresses.
```